From 53c46459dfaf25979c8063c899f55d43b82b15e3 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 3 Mar 2015 09:39:29 +0100
Subject: [PATCH] dracut.logrotate: remove yearly and increase size

In /etc/logrotate.d/dracut, as currently configured, the 'time' option
will always win out over the 'size' option.

https://bugzilla.redhat.com/show_bug.cgi?id=1005886
---
 dracut.logrotate | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/dracut.logrotate b/dracut.logrotate
index e7d4f0a..f4c35e0 100644
--- a/dracut.logrotate
+++ b/dracut.logrotate
@@ -1,7 +1,9 @@
 /var/log/dracut.log {
     missingok
     notifempty
-    size 30k
-    yearly
+    compress
+    delaycompress
+    copytruncate
+    size 1M
     create 0600 root root
 }
